doc/assets.md

```diff
@@ -10,41 +10,37 @@ For assets, a special **`ezdesign` asset package** is available.
 <img src="{{ asset("images/foo.png", "ezdesign") }}" alt="foo"/>
 ```
 
-Using `ezdesign` package will resolve current design with theme fallback.
+Using the `ezdesign` package will resolve current design with theme fallback.
 
 By convention, an asset theme directory can be located in:
-* `<bundle_directory>/Resources/public/themes/`
-* `web/assets/themes/`
+- `<bundle_directory>/Resources/public/themes/`
+- `web/assets/themes/`
 
 Typical paths can be for example:
-* `<bundle_directory>/Resources/public/themes/foo/` => Assets will be part of `foo` theme.
-* `<bundle_directory>/Resources/public/themes/bar/` => Assets will be part of `bar` theme.
-* `web/assets/themes/biz/` => Assets will be part of `biz` theme.
+- `<bundle_directory>/Resources/public/themes/foo/` => Assets will be part of the `foo` theme.
+- `<bundle_directory>/Resources/public/themes/bar/` => Assets will be part of the `bar` theme.
+- `web/assets/themes/biz/` => Assets will be part of the `biz` theme.
 
-It is also possible to use `web/assets` as a global override directory. 
-If called asset is present **directly under this directory**, it will always be considered first.
+It is also possible to use `web/assets` as a global override directory. If called asset is present **directly under this directory**, it will always be considered first.
 
-> **Important**: You must have *installed* your assets with `assets:install` command, so that your public resources are
-> *installed* into the `web/` directory.
+> **Important**: You must have *installed* your assets with `assets:install` command, so that your public resources are *installed* into the `web/` directory.
```

Comment:
Those lines are really long. Given that this change doesn't change anything to the rendered result, I suggest you leave it as it was.